tests/codebuild/feature_tests.sh

```diff
@@ -14,10 +14,32 @@ function run_feature_canary_tests
 function run_feature_integration_tests
 {
   echo "Running feature integration tests"
+  local crd_namespace="$1"
   run_feature_canary_tests
 
-  run_test testfiles/failing-xgboost-mnist-hpo.yaml
-  run_test testfiles/failing-xgboost-mnist-trainingjob.yaml
+  run_test "${crd_namespace}" testfiles/failing-xgboost-mnist-hpo.yaml
+  run_test "${crd_namespace}" testfiles/failing-xgboost-mnist-trainingjob.yaml
+}
+
+# Creates a training job in the specified namespace
+# For this test, the job is created in a namespace that does not have the operator. 
+function run_feature_namespaced_tests
+{
+  echo "Running feature namespaced tests"
+  local crd_namespace="$1"
+  run_feature_canary_tests
+  run_test "${crd_namespace}" testfiles/xgboost-mnist-trainingjob.yaml  
+}
+
+# Verifies that the job created in an incorrect namespace does not gain a status or sagemaker name.
+function verify_feature_namespaced_tests
+{
+  echo "Verifying namespace deployment test"
+  local crd_namespace="$1"
+  if ! verify_trainingjob_has_no_sagemaker_name "$crd_namespace" TrainingJob "xgboost-mnist"; then
```

Comment:
nit: Accept training job name as input - this is the standard we have been using.